src/main.env

```diff
@@ -27,7 +28,7 @@ AUTH_SERVER_IP=${MAIN_IP:-192.168.1.25}
 AUTH_SERVER_PORT=8081
 AUTH_SERVER_URL="http://${AUTH_SERVER_IP}:${AUTH_SERVER_PORT}"
 
-NGINX_HLS_SERVER_IP=${NGINX_RTMP_IP}
+NGINX_HLS_SERVER_IP=${EXTERNAL_IP}
```

Comment:
Maybe ${EXTERNAL_IP:-NGINX_RTMP_IP} to account for accessing site from local network

Reply:
![Screenshot from 2020-05-06 01-38-59](https://user-images.githubusercontent.com/21010314/81141412-8f1ae100-8f3a-11ea-81d0-22b5ad8e145f.png)
I believe you should set EXTERNAL_IP only if we're using cloud or something

Reply:
True, is there a way to detect if we are building the args locally or in production?

Reply:
I assume if this is in production then, the firewall would be down for a particular port. We should just check if that port is open publicly or not by pinging it I guess